src/Symfony/Component/DependencyInjection/ParameterBag/EnvPlaceholderParameterBag.php

```diff
@@ -94,9 +94,9 @@ public function resolve()
             if (!isset($this->parameters[$name = strtolower("env($env)")])) {
                 continue;
             }
-            if (is_numeric($default = $this->parameters[$name])) {
+            if (is_scalar($default = $this->parameters[$name])) {
```

Comment:
Can you explain the rationale... it already casts numbers to string. I assumed this should be any scalar; also null doesnt happen here due isset().

At this point cant we simply remove `EnvPlaceholderParameterBag::resolve()`?